js/views/pauseScreen.js

```diff
@@ -64,7 +65,8 @@ var PauseScreen = React.createClass({
   },
 
   handleTouchEnd: function(e) {
-    if (e.target.className === "oo-state-screen-selectable") {
+    var isSelectableScreen = this.props.areClassNamesEquals(e.target.className, this.selectableScreenClassName);
```

Comment:
Is it necessary to listen to these events at the document level even if we only execute the handler for a specific element? Wouldn't it be better to attach the event handler to that element only?

Reply:
For vr 360 we need to listen this event on document. And if add listener in two places it will be dublicate of the code.

Reply:
I see. I would personally prefer to have two separate handlers in order to avoid checking for a class (which could potentially change), but leave it up to the second reviewer @aeng7 @pilievOoyala

Reply:
yes, i separated it already in todays pr.